pkg_resources/__init__.py

```diff
@@ -1403,8 +1403,15 @@ def get_resource_string(self, manager, resource_name):
     def has_resource(self, resource_name):
         return self._has(self._fn(self.module_path, resource_name))
 
+    def get_metadata_path(self, name):
```

Comment:
This changes the public interface, do we really need to add a function to the public interface for this? I think I'd rather use a private method or do it in-line.